src/components/CallView/shared/LocalVideo.vue

```diff
@@ -187,17 +178,6 @@ export default {
 			return this.$store.getters.getDisplayName()
 		},
 
-		sessionHash() {
-			return Hex.stringify(SHA1(this.localCallParticipantModel.attributes.peerId))
-		},
-
-		guestName() {
-			return this.guestNameStore.getGuestName(
-				this.$store.getters.getToken(),
-				this.sessionHash,
-			)
-		},
```

Comment:
Conflict was that a sibling here was removed in https://github.com/nextcloud/spreed/pull/12913
So resolving the conflict was straight forward and I pushed it